CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/di/modules/OkHttpClientModule.kt

```diff
@@ -18,6 +23,10 @@ class OkHttpClientModule {
             addInterceptor(HttpLoggingInterceptor().apply {
                 level = HttpLoggingInterceptor.Level.BODY
             })
+            connectTimeout(CONNECT_TIME_OUT, TimeUnit.SECONDS)
+            writeTimeout(WRITE_TIME_OUT, TimeUnit.SECONDS)
```

Comment:
Could it be we don't need this? I'm thinking if `readTimeout` is sufficient 🙏

Reply:
You are correct, let me delete redundant time out
The `readTimeout` should be enough